app/services/exchange_rates/monthly_exchange_rates_service.rb

```diff
@@ -0,0 +1,38 @@
+module ExchangeRates
+  class MonthlyExchangeRatesService
+    class << self
+      def call
+        return unless tomorrow_is_penultimate_thursday? && today_is_wednesday?
+
+        ExchangeRates::UpdateCurrencyRatesService.new.call
+        ExchangeRates::UploadMonthlyFileService.call(:csv)
+        ExchangeRates::UploadMonthlyFileService.call(:xml)
+
+        notify
+      end
+
+      private
+
+      def notify
+        message = 'Exchange rates for the current month have been added and are accessible for viewing at /exchange_rates.'
+
+        logger.info message
+
+        SlackNotifierService.call(message)
+      end
+
+      def tomorrow_is_penultimate_thursday?
```

Comment:
Should we name this with the convention `is_`at the start?
`is_tomorrow_penultimate_thursday?` or
`is_tomorrow_the_penultimate_thursday?`